tests/data/test_boyd_study/extract_configs/Boyd_Minimal_Data_Elements_Cuellar_9-2-2018_EDITED_v2_map.py

```diff
@@ -0,0 +1,116 @@
+from common import constants
+from etl.extract.operations import *
+from etl.transform.standard_model.concept_schema import CONCEPT
+
+source_data_url = (
+    'file:///Users/kelmana/Documents/kids-first/kf-study-imports/Avi/'
```

Comment:
This should change to the S3 url for the file. I tried doing that but things didn't work and then I realized that we need #80 to merge first.

Reply:
lol yeah. I'm filing a bug. Trust me that it runs for now and I'll update it after these all merge.

Reply:
k 👍